dissect/target/plugins/os/windows/regf/shimcache.py

```diff
@@ -339,12 +343,12 @@ def shimcache(self) -> ShimcacheRecord:
                     self.target.log.warning("Error parsing ShimCache entry: %s %s", key, value_name)
                     continue
 
-                yield from self._get_records(cache)
+                yield from self._get_records(cache, value_name)
 
     def _get_records(
-        self, cache: Generator[ShimCacheGeneratorType, None, None]
+        self, cache, name: Generator[ShimCacheGeneratorType, None, None]
```

Comment:
```suggestion
        self, name: str, cache: Generator[ShimCacheGeneratorType, None, None]
```
Moving the order + fixing the type hint.